### PR TITLE
daily status reminder hubot

### PR DESCRIPTION
**Pull request by nikolaiwarner**

### CODE REVIEW

File: scripts/status_reminder.coffee [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#   Remind users to give a daily status update
+#
+#   You can use package hubot-cron-json to execute the send reminders task
+#   at a certain time. Update conf/cron-tasks.json to set the time.
+#
+# Commands:
+#   status reminder add user <username> - Add a user
+#   status reminder remove user <username> - Remove a user
+#   status reminder list users - List all users getting reminders and show stats
+#   status reminder add user <username> - Add a user
```
Comment (dhempy) [inline scripts/status_reminder.coffee]: Duplicate.

File: scripts/status_reminder.coffee [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      msg.send "#{user.username} - Streak: #{user.streak} - Last update: #{new Date(user.last_status_date)}"
+
+  robot.respond /status reminder send reminders/i, ->
+    send_reminders()
+
+  robot.on /status reminder send reminders/i, ->
+    send_reminders()
+
+  robot.hear /^t:|^today:/i, (msg) ->
+    username = msg.message.user.name
```
Comment (nuex) [inline scripts/status_reminder.coffee]: It will be interesting to see if this is bot compatible. I don't want to join #standups.

**Comment by nuex** [comment]
Haven't cloned and tested this, but it looks solid.

+1

**Comment by dhempy** [comment]
Seems like everyone should join #standups.

**Pull request merged**